example-2/evm/src/contracts/lendingHub/HubGetters.sol

```diff
@@ -83,6 +83,10 @@ contract HubGetters is Context, HubStructs, HubState {
         return _state.interestAccrualIndexPrecision;
     }
 
+    function getMaxDecimals() internal view returns (uint8) {
+        return _state.MAX_DECIMALS;
```

Comment:
i will update this to be an element of state that is updated whenever register happens